modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -320,7 +327,9 @@ private static boolean isVoid(Class<?> type)
 
     public static Collection<Method> getOperationMethods(Class<?> declaringClass)
     {
-        return getAllMethods(declaringClass, withAnnotation(Operation.class), withModifier(Modifier.PUBLIC));
+        Set<Method> methods = getAllMethods(declaringClass, withModifier(Modifier.PUBLIC));
```

Comment:
inefficient. Replace with return getAllMethods(declaringClass, withModifier(Modifier.PUBLIC), Predicates.not(withAnnotation(Ignore.class)));